src/main/java/com/coffee/gifu/service/util/apiFilter.java

```diff
@@ -0,0 +1,5 @@
+package com.coffee.gifu.service.util;
+
+public class apiFilter {
```

Comment:
**### Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'.**